packages/flutter/lib/src/material/tabs.dart

```diff
@@ -826,7 +827,7 @@ class _TabBarState extends State<TabBar> {
       color = Colors.white;
 
     return UnderlineTabIndicator(
-      insets: widget.indicatorPadding,
+//      insets: widget.indicatorPadding,
```

Comment:
remove this line?

Reply:
@chunhtai Throwing a FlutterError if `(rect.size >= insets.collapsedSize)` isnt true.